proto/ibc/applications/middleware/fee/v1/query.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "google/api/annotations.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Query provides defines the gRPC querier service.
+service Query {
+  // Gets the fee expected for submitting ReceivePacket msg for the given packet
+  rpc ReceiveFee(QueryReceiveFeeRequest) returns (QueryReceiveFeeResponse) {
+    option (google.api.http).get = "/ibc/apps/middleware/fee/v1/receive_fee";
+  }
+
+  // Gets the fee expected for submitting AcknowledgePacket msg for the given packet
+  rpc AckFee(QueryAckFeeRequest) returns (QueryAckFeeResponse) {
+    option (google.api.http).get = "/ibc/apps/middleware/fee/v1/ack_fee";
+  }
+
+  // Gets the fee expected for submitting TimeoutPacket msg for the given packet
+  rpc TimeoutFee(QueryTimeoutFeeRequest) returns (QueryTimeoutFeeResponse) {
+    option (google.api.http).get = "/ibc/apps/middleware/fee/v1/timeout_fee";
+  }
+}
+
+// QueryReceiveRequestFee is the request type for quering the receive fee
+message QueryReceiveFeeRequest {
+  // Source Port ID
+  string port_id = 1 [(gogoproto.moretags) = "yaml:\"port_id\""];
+  // Source Channel ID
+  string channel_id = 2 [(gogoproto.moretags) = "yaml:\"channel_id\""];
+  // Packet sequence
+  string sequence = 3 [(gogoproto.moretags) = "yaml:\"sequence\""];
```

Comment:
yaml is only needed when we have an underscore. The go proto compiler defaults to camelcase, so we need to add the underscore when we have two words

Reply:
gotcha. updated.